quasar-cli-vue3-webpack-javascript/e2e/CheckAppExposes.cy.ts

```diff
@@ -0,0 +1,302 @@
+import {BaseMethods} from "../../cypress/common/base";
```

Comment:
Try to think more how you can simplify this test) I think there is route to make it more smaller)